tools/ansible/tasks/wsu/main.yaml

```diff
@@ -151,3 +162,27 @@
       until: '"k8s.ovn.org/hybrid-overlay-distributed-router-gateway-mac" in overlay.stdout'
       retries: 12
       delay: 5
+
+    - name: Get hybrid overlay subnet
+      delegate_to: localhost
+      shell: oc get nodes "{{ node_name.stdout }}" -o=jsonpath='{.metadata.annotations.k8s\.ovn\.org/hybrid-overlay-hostsubnet}'
+      register: hybrid_subnet
+
+    - name: Create cni config directory
```

Comment:
I am trying to address this here - https://github.com/openshift/windows-machine-config-operator/pull/109